streaming/flink/flink-team.tf

```diff
@@ -0,0 +1,107 @@
+locals {
+  flink_team = "flink-team-a"
+}
+
+#---------------------------------------------------------------
+# Creates IAM policy for IRSA. Provides IAM permissions for flink pods
+#---------------------------------------------------------------
+resource "aws_iam_policy" "flink" {
+  description = "IAM role policy for flink Job execution"
+  name        = "${local.name}-flink-irsa"
+  policy      = data.aws_iam_policy_document.flink_operator.json
+}
+
+#---------------------------------------------------------------
+# IRSA for flink pods for "flink-team-a"
+#---------------------------------------------------------------
+module "flink_irsa" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/irsa?ref=v4.15.0"
```

Comment:
Can we use `github.com/aws-ia/terraform-aws-eks-blueprints-addons//modules/irsa`  instead?

Reply:
updated